Makefile

```diff
@@ -0,0 +1,103 @@
+PWD := $(shell pwd)
+LOCAL_BIN ?= $(PWD)/bin
+
+# ENVTEST_K8S_VERSION refers to the version of kubebuilder assets to be downloaded by envtest binary.
+ENVTEST_K8S_VERSION = 1.29.x
+
+# Keep an existing GOPATH, make a private one if it is undefined
+GOPATH_DEFAULT := $(PWD)/.go
+export GOPATH ?= $(GOPATH_DEFAULT)
+GOBIN_DEFAULT := $(GOPATH)/bin
+export GOBIN ?= $(GOBIN_DEFAULT)
+export PATH := $(LOCAL_BIN):$(GOBIN):$(PATH)
+
+include build/common/Makefile.common.mk
+
+############################################################
+# clean section
+############################################################
+
+.PHONY: clean
+clean:
+	-rm bin/*
+	-rm -r vendor/
+	-rm build/_output/*
+	-rm kubeconfig_$(CLUSTER_NAME)_e2e
+	kind delete cluster --name test-$(CLUSTER_NAME)
+
+############################################################
+# build section
+############################################################
+BUILD_DIR := $(PWD)/build/_output
+BINARY_DIR ?= $(PWD)/cmd/*
+BINARY_NAME := policytools
+CONTAINER_ENGINE ?= podman
+# Parse the version using git, with fallbacks as follows:
+# - git describe (i.e. vX.Y.Z-<extra_commits>-<sha>)
+# - <branch>-<sha>
+# - <sha>-dev
+# - Go BuildInfo version
+# - Unversioned binary
+GIT_VERSION := $(shell git describe --dirty 2>/dev/null)
+ifndef GIT_VERSION
+  GIT_BRANCH := $(shell git branch --show-current)
+  GIT_SHA := $(shell git rev-parse --short HEAD)
+  ifdef GIT_BRANCH
+    GIT_VERSION := $(GIT_BRANCH)-$(GIT_SHA)
+  else ifdef GIT_SHA
+    GIT_VERSION := $(GIT_SHA)-dev
+  endif
+endif
+GO_LDFLAGS ?= -X 'main.Version=$(GIT_VERSION)'
```

Comment:
Is this used for anything right now? I guess it would be used in either help or version outputs, but unless Cobra reads this automagically, I don't see how this is connected to anything. We can keep it if there's a plan for it, I just wanted to check if I was missing anything